src/test/java/foodwhere/storage/JsonAdaptedStallTest.java

```diff
@@ -13,84 +13,66 @@
 import foodwhere.commons.exceptions.IllegalValueException;
 import foodwhere.model.stall.Address;
 import foodwhere.model.stall.Name;
-import foodwhere.model.stall.Phone;
 
 public class JsonAdaptedStallTest {
     private static final String INVALID_NAME = "R@chel";
     private static final String INVALID_PHONE = "+651234";
     private static final String INVALID_ADDRESS = " ";
-    private static final String INVALID_DETAIL = "#friend";
+    private static final String INVALID_EMAIL = "example.com";
+    private static final String INVALID_TAG = "#friend";
 
     private static final String VALID_NAME = BENSON.getName().toString();
     private static final String VALID_PHONE = BENSON.getPhone().toString();
     private static final String VALID_ADDRESS = BENSON.getAddress().toString();
-    private static final List<JsonAdaptedDetail> VALID_DETAILS = BENSON.getDetails().stream()
+    private static final List<JsonAdaptedDetail> VALID_TAGS = BENSON.getDetails().stream()
```

Comment:
Perhaps it would be better to leave it as DETAILS as `Tag` is changed to `Detail` in earlier commits?

Reply:
Resolved this by keeping it as `DETAILS`.